RandomData.Api/StringGeneration/ServiceImplementations/RandomStringGenerationService.cs

```diff
@@ -0,0 +1,36 @@
+﻿using RandomData.Api.Services.Random;
+using RandomData.Api.StringGeneration.Dto;
+using RandomData.Api.StringGeneration.Exceptions;
+using RandomData.Api.StringGeneration.Extensions;
+using RandomData.Api.StringGeneration.Validators;
+
+namespace RandomData.Api.StringGeneration.ServiceImplementations
+{
+    public class RandomStringGenerationService : IStringGenerationService
+    {
+        private readonly IRandom _random;
+        private readonly GetStringParametersValidator _validator = new GetStringParametersValidator();
+
+        public RandomStringGenerationService(IRandom random)
+        {
+            _random = random;
+        }
+
+        public string GenerateRandomString(GetStringParameters parameters)
```

Comment:
If you group line of codes in groups by adding empty lines it's probably information that you should split this method to smaller.

Reply:
I think that the length of this method is ok and there is no need to split it more. I would even say that it wouldn't improve a lot. However if you really think it should be changed I'll of course do it.